README.Rmd

```diff
@@ -40,6 +40,8 @@ If you are new to dplyr, the best place to start is the [data transformation cha
 
 In addition to data frames/tibbles, dplyr makes working with other computational backends accessible and efficient. Below is a list of alternative backends:
 
+- [arrow](https://arrow.apache.org/docs/r) for larger-than-memory datasets with two backend engine options (default engine is [Acero](https://arrow.apache.org/docs/cpp/streaming_execution.html), the Arrow C++ engine, or [DuckDB](https://duckdb.org/))
```

Comment:
I feel like the duckdb integration is too subtle to mention in this bullet point--duckdb is just another dbplyr backend, we've (us and the duckdb developers) just made it so you can stream data between Acero and duckdb. I wouldn't describe `arrow` as having two different backends: it provides an Arrow backend for dplyr, and facilities for integrating with others. The plan is that before long, duckdb won't be exceptional in that regard--that's what ADBC is about. 

How about something like this? I also wonder whether introducing "Acero" here is distracting since no one (yet) knows the name.

```suggestion
- [arrow](https://arrow.apache.org/docs/r) for larger-than-memory datasets, including on remote cloud storage like AWS S3, using the Apache Arrow C++ engine, [Acero].(https://arrow.apache.org/docs/cpp/streaming_execution.html)
```